editor/plugins/camera_3d_editor_plugin.cpp

```diff
@@ -79,37 +82,69 @@ Camera3DEditor::Camera3DEditor() {
 	preview->connect("pressed", callable_mp(this, &Camera3DEditor::_pressed));
 }
 
+static Size2i _get_viewport_size(Camera3D *p_camera) {
```

Comment:
This function can be inside `_update_sub_viewport_size()`, because it's used only once.